src/AppInstallerCLICore/ExecutionContextData.h

```diff
@@ -190,6 +191,12 @@ namespace AppInstaller::CLI::Execution
             using value_t = std::vector<std::tuple<Utility::LocIndString, Utility::LocIndString, Utility::LocIndString>>;
         };
 
+        template <>
+        struct DataMapping<Data::ProductCodeFromARP>
+        {
+            using value_t = std::vector<Utility::LocIndString>;

```

Comment:
This needs to be the entire `AppsAndFeaturesEntry`, not just the product code.

Reply:
Updated to include the whole AppAndFeaturesEntry, although I'm not entirely sure if I did it The Right Way®